pheps/phep-0001.md

```diff
@@ -0,0 +1,360 @@
+
+# PHEP Applicability
+<a name="phep-applicability"></a>
+**Standards** Track PHEPs are recommended standards for PyHC packages. Most packages should make a good-faith effort to comply. PyHC core packages are strongly recommended to comply and generally must do so, absent a compelling reason not to. A PHEP which core packages cannot agree to implement likely does not have sufficient community support for acceptance. Standards track PHEP compliance must be considered as a factor in evaluating current and potential PyHC packages.
+
+**Informational** PHEPs are for the benefit of PyHC developers and users and may represent "best practices". They are not binding.
+
+**Process** PHEPs determine the process by which the PyHC community makes formal decisions and as such bind all participants in PyHC.
+
+For all three types, key words in PHEPs must be interpreted according to [RFC2119](https://datatracker.ietf.org/doc/html/rfc2119); the use of `must`, `must not`, `should`, and `should not` are preferred.
+
+# PHEP Workflow
+<a name="phep-workflow"></a>
+ All PHEP workflow is conducted via the [GitHub PyHC standards repository](https://github.com/heliophysicsPy/standards).
+
+## PHEP Editors
+<a name="phep-editors"></a>
+PHEP editors are responsible for managing the administrative and editorial aspects of the PHEP workflow (e.g. assigning PHEP numbers and changing their status). See [PHEP Editor Responsibilities & Workflow](#phep-editor-responsibilities-workflow) for details.
+
+PHEP editorship is by nomination of the current editors, subject to approval by PyHC leadership. In the event there are fewer than two active editors, PyHC leadership may appoint editors from active participants in the community, sufficient to bring the total to two.
+
+The editors can be contacted by mentioning `@heliophysicsPy/phep-editors` on GitHub.
+
+## Start with an idea
+<a name="start-with-an-idea"></a>
+The PHEP process begins with a new idea for PyHC. It is highly recommended that a single PHEP contain a single key proposal or new idea; the more focused the PHEP, the more successful it tends to be. If in doubt, split your PHEP into several well-focused ones.
+
+Each PHEP must have an author: someone who writes the PHEP using the style and format described below, shepherds the discussion, and attempts to build community consensus around the idea. The PHEP author should first attempt to ascertain whether the idea is PHEP-able. Suggested venues for this discussion include a PyHC [telecon](https://heliopython.org/meetings/), the PyHC [mailing list](https://heliopython.org/contact/), PyHC [Matrix/Slack](https://riot.im/app/#/room/#heliopython:openastronomy.org), and [an issue in the standards repository](https://github.com/heliophysicsPy/standards/issues/new/choose) (which should be linked to the PR when the PHEP is submitted). The author may choose the approach that seems best for their topic.
```

Comment:
```suggestion
Each PHEP must have an author: someone who writes the PHEP using the style and format described below, shepherds the discussion, and attempts to build community consensus around the idea. The PHEP author should first attempt to ascertain whether the idea is PHEP-able. Suggested venues for this discussion include a PyHC [telecon](https://heliopython.org/meetings/), the PyHC [mailing list](https://heliopython.org/contact/), PyHC [Matrix/Slack](https://app.element.io/#/room/#heliopython:openastronomy.org), and [an issue in the standards repository](https://github.com/heliophysicsPy/standards/issues/new/choose) (which should be linked to the PR when the PHEP is submitted). The author may choose the approach that seems best for their topic.
```

Reply:
Thanks for the catch and easy fix. I was working off the contact page so I opened heliophysicsPy/heliophysicsPy.github.io#304 to fix that.

I wonder if it makes more sense to just keep this to linking the contact page instead of element directly, as I did for the mailing list. That way stuff can move around without needing to update here.

Reply:
Ooops, latest push is still directly linking chatroom instead of the contact page. I'll let this sit for a bit.

Reply:
I like the idea of switching to the Contact page, so people can "choose their own adventure" w.r.t. which chat to use.

Reply:
[Just linking the contact page now](https://github.com/heliophysicsPy/standards/blob/68125caadd7c081ecc8fc9f803c6d3ebebebce58/pheps/phep-0001.md?plain=1#L71) with the idea that if we add more means of communication there, there's no need to update this.